src/grappler/ngraph_optimizer.cc

```diff
@@ -62,15 +86,17 @@ Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
   }
 
   // If ngraph is disabled via ngraph_bridge api or NGRAPH_TF_DISABLE is set
-  // we will not do anything; all subsequent
-  // passes become a no-op.
+  // we will not do anything; all subsequent passes become a no-op.
   bool ngraph_not_enabled =
       (!config::IsEnabled()) || (std::getenv("NGRAPH_TF_DISABLE") != nullptr);
   bool already_processed = IsProcessedByNgraphPass(&graph);
+  if (!already_processed && ngraph_not_enabled) {
+    NGRAPH_VLOG(0) << "Graph will not run through ngraph";
```

Comment:
Since this is a VLOG(0) level message - need to be a bit more user friendly. Need to specify why the graph won't run through nGraph (think of a message that an user would get and wonder - what can I do to fix/change this). 
So - use something like: `nGraph is available but disabled`. 
If the graph is alreasy transformed ti nGraph then it won't be further transformed but `WILL run` through nGraph. So this message `Graph will not run through ngraph` is misleading - no?

Reply:
`!already_processed && ngraph_not_enabled` implies will not run thru nGraph
Its not processed AND ngraph is disabled, so no chance of forming encapsulates. So no running thru nGraph.


Comment about helpfulness of message stands

Reply:
How about `nGraph is disabled, so graph will not be transformed`.